internal/mount/flag.go

```diff
@@ -31,6 +33,24 @@ const (
 	// stat-cache-ttl or type-cache-ttl if they have not been set
 	// by the user.
 	DefaultStatOrTypeCacheTTL time.Duration = time.Minute
+	// DefaultStatCacheCapacity is the default value for stat-cache-capacity.
+	// This is never really used by GCSFuse for configuring stat-cache size.
+	// It is used only to determine if the user has explicitly set a value of stat-cache-capacity.
+	DefaultStatCacheCapacity = math.MinInt
```

Comment:
we have to use a value corresponding to 32MiB here, otherwise it will look odd to the user when they do gcsfuse --help

Reply:
done